x-pack/legacy/plugins/siem/package.json

```diff
@@ -10,18 +10,8 @@
     "cypress:run": "cypress run --spec ./cypress/integration/**/*.spec.ts --reporter mocha-multi-reporters --reporter-options configFile=./reporter_config.json; mochawesome-merge --reportDir ../../../../target/kibana-siem/cypress/results > ../../../../target/kibana-siem/cypress/results/output.json; marge ../../../../target/kibana-siem/cypress/results/output.json --reportDir ../../../../target/kibana-siem/cypress/results; mkdir -p ../../../../target/junit && cp ../../../../target/kibana-siem/cypress/results/*.xml ../../../../target/junit/"
   },
   "devDependencies": {
-    "@cypress/webpack-preprocessor": "^4.1.0",
-    "@types/js-yaml": "^3.12.1",
     "@types/lodash": "^4.14.110",
-    "@types/react-beautiful-dnd": "^10.0.1",
-    "cypress": "^3.4.1",
-    "js-yaml": "^3.13.1",
-    "mocha-junit-reporter": "^1.23.1",
-    "mocha-multi-reporters": "^1.1.7",
-    "mochawesome": "^4.0.1",
-    "mochawesome-merge": "^2.0.1",
-    "mochawesome-report-generator": "^4.0.1",
-    "ts-loader": "^6.0.4"
```

Comment:
Looks like all of these packages (with the exception of `@types/js-yaml` which I have re-added to siem/package.json) have the same versions in x-pack/package.json. Only difference is `ts-loader` is located in the `dependencies` and not `devDependencies` in x-pack/package.json.